# --- Time params
# Origin (for real car)
# Tp    : 3.0    # prediction horizon [s]
# Ts_MPC: 0.05    # MPC prediction discretization step [s]

# MORAI (for real car)
Tp    : 3.0    # prediction horizon [s]
Ts_MPC: 0.032    # MPC prediction discretization step [s]

# --- Controller
# MPC_type : 'NMPC'
MPC_param_file : 'ioniq5/MPC_params.yaml'
# --- Model
model_switch                    : False
MPC_switch_model                : 'kinematic_stm'
switch_model_costfunction_type  : 'NONLINEAR_LS'

# MPC_model                       : 'dynamic_stm_pacejka'
MPC_model                       : 'kinematic_stm'
MPC_model_costfunction_type     : 'NONLINEAR_LS'

veh_params_file_MPC   : "ioniq5/veh_params_pred.yaml"
tire_params_file_MPC  : "ioniq5/pacejka_params.yaml"

# --- Fallback params